clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/ReservationReportCachingAgent.groovy

```diff
@@ -268,7 +279,13 @@ class ReservationReportCachingAgent implements CachingAgent, CustomScheduledAgen
         def describeInstancesRequest = new DescribeInstancesRequest().withMaxResults(500)
         def allowedStates = ["pending", "running"] as Set<String>
         while (true) {
-          def result = amazonEC2.describeInstances(describeInstancesRequest)
+          DescribeInstancesResult result
+          try {
```

Comment:
is there more that could fail on a region than just this call? should the whole flatMap per region be try-catch'd